UI/Authentication.OidcDemo/Authentication.OidcDemo/Authentication.OidcDemo.Droid/Main.cs

```diff
@@ -27,11 +27,11 @@ public class Application : Windows.UI.Xaml.NativeApplication
 		public Application(IntPtr javaReference, JniHandleOwnership transfer)
 			: base(() => new App(), javaReference, transfer)
 		{
-			ConfigureUniversalImageLoader();
-
 			ApiExtensibility.Register(
 				typeof(IWebAuthenticationBrokerProvider),
 				_ => new ChromeCustomTabsProvider());
+
+			ConfigureUniversalImageLoader();
```

Comment:
I'm curious why you change this order... I can't see how it can have an impact on the app.

Reply:
I will double-check the behavior and let will know. Thanks!